javascript.md

```diff
@@ -283,7 +336,8 @@ console.log(list.length); // outputs 3
 Rust:
 
 ```rust
-// TODO
+let list = vec![1, 2, 3];
```

Comment:
It denotes a macro. The standard library provides some macros like `format!` and `vec!`, and [one can define one's own macros :)](https://doc.rust-lang.org/stable/book/macros.html)